packages/math-input/src/components/tabbar/item.tsx

```diff
@@ -75,70 +76,146 @@ function imageTintColor(
     return color.offBlack64;
 }
 export type ItemState = "active" | "inactive" | "disabled";
-type Props = {
+type ArrowKeyTabItemProps = {
     onClick: () => void;
     itemState: ItemState;
     itemType: KeypadPageType;
+    focus: boolean;
+    setFocus: React.Dispatch<React.SetStateAction<number>>;
```

Comment:
I wonder if we could simplify this to just `(index: number) => void` and wrap the setter when providing the prop.

Here:

```suggestion
    setFocus: (index: number) => void;
```

In `tabbar.tsx`:

``` JSX
                       setFocus={(index: number) => setFocus(index)}
```

Reply:
I was getting some weird errors when attempting to do just this. But let me try again 😉